### PR TITLE
Sands of Dorne first preview

### PR DESCRIPTION
**Pull request by DukeTax**
@Alsciende: I'm not entirely sure whether the pack code for this expansion should be 10 or 11. So far, one box has released within each cycle, establishing the pattern of cycle-box-cycle-box-etc. But since FFG is accelerating the release of boxes compared to cycles, this expansion will likely release before the next cycle. I have opted to use 10 for now, which conforms to (likely) release order, but breaks the earlier established pattern. Let me know if this is incorrect.

Side note: House of Thorns (09) was also released before the first pack of cycle 4 (08).

### CODE REVIEW

File: pack/SoD.json [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        "is_military": false,
+        "is_power": false,
+        "is_unique": true,
+        "name": "Tyene Sand",
+        "octgn_id": null,
+        "pack_code": "SoD",
+        "position": 8,
+        "quantity": 3,
+        "strength": 3,
+        "text": "Stealth\n<b>Reaction:</b> After you lose a challenge as the defending player, kneel a <i>Sand Snake</i> character to put Tyene Sand into play from your hand.",
```
Comment (ystros) [inline pack/SoD.json]: Missing period after Stealth.

File: pack/SoD.json [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        "pack_code": "SoD",
+        "position": 39,
+        "quantity": 3,
+        "strength": null,
+        "text": "<b>Reaction:</b> After Val is declared as an attacker, put a <i>Wildling</i> character with printed cost 4 or lower into play from your hand knelt, participating as an attacker.",
+        "traits": "Wildling.",
+        "type_code": "character"
+    },
+    {
+        "code": "09045",
```
Comment (ystros) [inline pack/SoD.json]: 10045, assuming cycle code 10 is going to be used.

File: pack/SoD.json [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+        "initiative": 0,
+        "is_loyal": true,
+        "is_unique": false,
+        "name": "At Prince Doran's Behest",
+        "octgn_id": null,
+        "pack_code": "SoD",
+        "position": 46,
+        "quantity": 3,
+        "reserve": 0,
+        "text": "<b>When Revealed:</b> Choose and reveal a new plot card. (Cannot be cancelled.)",
```
Comment (ystros) [inline pack/SoD.json]: Spelling on card is "canceled" (1 L)

**Comment by Alsciende** [comment]
@DukeTax position 10 is good. Thank you!

**Pull request merged**